### DISCUSSION TITLE
API Stats Query for Pageviews per Page

### DISCUSSION THREAD

**Discussion by dknight10** [plausible/analytics#1069] [Self-Hosted Support] [answered]
Is there a query using the stats API that will fetch the page views per page. I can get the visitors per page using this query: 
`https://plausible.io/api/v1/stats/breakdown?site_id={site_id}&period=7d&property=event:page`

But have not found a way to get the page views.

**Comment by ukutaht** [comment, marked as answer]
Hey! Try `&metrics=pageviews` parameter.

**Reply by dknight10** [reply]
Perfect. Thank you!
